pkg/server/handler/tikvhandler/tikv_handler.go

```diff
@@ -1785,45 +1785,66 @@ type DBTableInfo struct {
 
 // ServeHTTP handles request of database information and table information by tableID.
 func (h DBTableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	schema, err := h.Schema()
+	if err != nil {
+		handler.WriteError(w, err)
+		return
+	}
+
+	if tableIDsStr := req.FormValue(handler.TableIDsQuery); len(tableIDsStr) > 0 {
+		tableIDs := strings.Split(tableIDsStr, ",")
+		data := make(map[int]*DBTableInfo, len(tableIDs))
+		for _, tableID := range tableIDs {
+			tid, err := strconv.Atoi(tableID)
+			if err != nil {
+				handler.WriteError(w, err)
+				return
+			}
+			tbl, err := getDBTableInfo(schema, int64(tid))
+			if err == nil {
+				data[tid] = tbl
+			}
+		}
+		handler.WriteData(w, data)
```

Comment:
maybe consider wrap data inside another common struct for further extend
```
{
   data: <real-data>
}
```

Reply:
@mornyx Do you think it's better we add another level JSON object?

Reply:
`/schema` does not follow this design. Maybe we should keep them consistent?